### PR TITLE
Bug fix/wake vortex

### PR DESCRIPTION
**Pull request by roger-teoh**
Closes #XX

## Changes

Address unit inconsistency in `turbulent_kinetic_energy_dissipation_rate function` as identified by Wessel99, and update `wake_vortex.py`

#### Fixes
- Bug fix in `turbulent_kinetic_energy_dissipation_rate` in `wake_vortex.py`
- Improve docstrings in `wake_vortex.py`
- Rename functions in `wake_vortex.py` by removing "get" at the start of each function. 

## Tests

- [ ] QC test passes locally (`make test`)
- [ ] CI tests pass

## Reviewer

> Select @ reviewer

### CODE REVIEW

**Comment by roger-teoh** [comment]
@zebengberg You might need to update the .json files in the unit testing of CoCiP outputs.

File: pycontrails/models/cocip/wake_vortex.py [hunk 1]
```diff
@@ -260,17 +262,22 @@ def turbulent_kinetic_energy_dissipation_rate(
...
-    See eq. (37) in :cite:`schumannContrailCirrusPrediction2012`.
+    - See eq. (37) in :cite:`schumannContrailCirrusPrediction2012`.
+    - In a personal correspondence, Schumann identified a print error in Eq. (37) of the 2012 paper
+        where the shear term should not be squared.
+    - The correct equation is listed in Eq. (13) of Schumann & Gerz (1995)
 
     References
     ----------
     - :cite:`schumannContrailCirrusPrediction2012`
+    - Schumann, U. and Gerz, T., 1995. Turbulent mixing in stably stratified shear flows.
```
Comment (mlshapiro) [inline pycontrails/models/cocip/wake_vortex.py:277]: I will add this to the bibliography on your branch

**Review by mlshapiro — APPROVED** [review]
Thanks Roger! I've reviewed, and it looks good. Thanks for the documentation improvements.

I'll add the new reference to the bibliography on Zotero and make the citation.

@zebengberg I'll merge this into main once I'm done.

**Comment by zebengberg** [comment]
Great work @roger-teoh!

@mlshapiro: can you also update the changelog? If the static cocip test files need to be significantly updated, I think you should cut a new release (v0.45.1).

**Pull request merged**